src/drivers/adc/adc.h

```diff
@@ -0,0 +1,17 @@
+#ifndef ADC_H
+#define ADC_H
+
+#include <stdbool.h>
+

```

Comment:
nit: add `#include "rocketlib/includes/common.h"` since u used w_status_t

idk why omitting it didnt throw a compile warning.. i will look into that.